packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -1626,6 +1626,7 @@ export class AppLocalGetEx extends Op {
       stack.push(BIGINT1);
     } else {
       stack.push(BIGINT0); // The value is zero if the key does not exist.
+      stack.push(BIGINT0); // did_exist_flag
```

Comment:
Why adding another zero? According to spec, we should push only one `0`.

Reply:
Yes, i thought the same, that's why we were previously pushing only one `0`. But i confirmed this using the `teal-debugger`, it pushes two `0's` to stack (if value is not found)

Reply:
Maybe there is a bug in the debugger. Let's validate on Discord with Algorand Team. Clearly there is something wrong: either in debugger or in docs.

Reply:
I was reading the https://developer.algorand.org/docs/reference/teal/opcodes/#app_local_get_ex
Yes, it will put two values.

We also need to update the doc string here.

Reply:
Alright, updating doc string.